drivers/stmpe811/include/stmpe811_constants.h

```diff
@@ -0,0 +1,229 @@
+#define STMPE811_TSC_CTRL_TRACK_4                   (0b001)     /**< Tracking index 4 */
+#define STMPE811_TSC_CTRL_TRACK_8                   (0b010)     /**< Tracking index 8 */
+#define STMPE811_TSC_CTRL_TRACK_16                  (0b011)     /**< Tracking index 16 */
+#define STMPE811_TSC_CTRL_TRACK_32                  (0b100)     /**< Tracking index 32 */
+#define STMPE811_TSC_CTRL_TRACK_64                  (0b101)     /**< Tracking index 64 */
+#define STMPE811_TSC_CTRL_TRACK_96                  (0b110)     /**< Tracking index 96 */
+#define STMPE811_TSC_CTRL_TRACK_127                 (0b111)     /**< Tracking index 127 */
+#define STMPE811_TSC_CTRL_STA                       (1 << 7)    /**< Touchscreen status (1: touch detected, 0: no touch detected), read-only */
+/** @} */
+
+/* @name    TSC_CFG register bitfields
+ * @{
+ */
+#define STMPE811_TSC_CFG_AVE_CTRL_POS               (6)         /**< Average control bit shift position */
+#define STMPE811_TSC_CFG_AVE_CTRL_1                 (0b00)      /**< Average control, 1 sample */
+#define STMPE811_TSC_CFG_AVE_CTRL_2                 (0b01)      /**< Average control, 2 sample */
+#define STMPE811_TSC_CFG_AVE_CTRL_4                 (0b10)      /**< Average control, 4 sample */
+#define STMPE811_TSC_CFG_AVE_CTRL_8                 (0b11)      /**< Average control, 8 sample */
+#define STMPE811_TSC_CFG_TOUCH_DET_DELAY_POS        (3)         /**< Touch detection delay bit shift position */
+#define STMPE811_TSC_CFG_TOUCH_DET_DELAY_10US       (0b000)     /**< Touch detection 10us delay */
+#define STMPE811_TSC_CFG_TOUCH_DET_DELAY_50US       (0b001)     /**< Touch detection 50us delay */
+#define STMPE811_TSC_CFG_TOUCH_DET_DELAY_100US      (0b010)     /**< Touch detection 100us delay */
+#define STMPE811_TSC_CFG_TOUCH_DET_DELAY_500US      (0b011)     /**< Touch detection 500us delay */
+#define STMPE811_TSC_CFG_TOUCH_DET_DELAY_1MS        (0b100)     /**< Touch detection 1ms delay */
+#define STMPE811_TSC_CFG_TOUCH_DET_DELAY_5MS        (0b101)     /**< Touch detection 5ms delay */
+#define STMPE811_TSC_CFG_TOUCH_DET_DELAY_10MS       (0b110)     /**< Touch detection 10ms delay */
+#define STMPE811_TSC_CFG_TOUCH_DET_DELAY_50MS       (0b111)     /**< Touch detection 50ms delay */
+#define STMPE811_TSC_CFG_SETTLING_10US              (0b000)     /**< Settling time 10us */
+#define STMPE811_TSC_CFG_SETTLING_50US              (0b001)     /**< Settling time 50us */
+#define STMPE811_TSC_CFG_SETTLING_100US             (0b010)     /**< Settling time 100us */
+#define STMPE811_TSC_CFG_SETTLING_500US             (0b011)     /**< Settling time 500us */
+#define STMPE811_TSC_CFG_SETTLING_1MS               (0b100)     /**< Settling time 1ms */
+#define STMPE811_TSC_CFG_SETTLING_5MS               (0b101)     /**< Settling time 5ms */
+#define STMPE811_TSC_CFG_SETTLING_10MS              (0b110)     /**< Settling time 10ms */
+#define STMPE811_TSC_CFG_SETTLING_50MS              (0b111)     /**< Settling time 50ms */
```

Comment:
`STMPE811_TSC_CFG_SETTLING_50US` doesn't exist in datasheet, so the following defines are shift.

Reply:
Fixed